notebooks/MNIST-torchvision.ipynb

```diff
@@ -0,0 +1,798 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x7f2d398af748>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plot_example(X_pred, y_pred[error_mask])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Convolutional Network\n",
+    "PyTorch expects a 4 dimensional tensor as input for its 2D convolution layer. The dimensions represent:\n",
```

Comment:
This needs an introductory sentence. Maybe: "Next we want to use a convolutional neural network instead of a simple feedforward netowrk."